spec/opensearch/api/actions/remote_store/restore_spec.rb

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
```

Comment:
The modifications header is not needed with only the above. It's for code that came from APLv2 Elastic that had `Licensed to Elasticsearch B.V. under ...`.